### ISSUE TITLE
Add requirements.txt file

### ISSUE THREAD

**Issue by orduek** [LevyDecisionNeuroLab/pipeline#1]
Add requierments.txt file, so people will have an easy time installing a conda environment.

**Comment by cherruc** [comment]
Done!

**Comment by cherruc** [comment]
https://github.com/LevyDecisionNeuroLab/pipeline/blob/6bb3780751dfc19d3c82fa23f3ad4863a92fbe78/requirements.txt

**Comment by orduek** [comment]
Great! Thanks, Chelsea.
How did you generated the file?

**Comment by cherruc** [comment]
I accessed the Farnam shell and generated the file with the following commands:
```
module load miniconda
source info --env
source activate py_37
conda list
conda list -e > requirements.txt
```

Apparently Google says the requirements.txt file can be installed into a new project environment with:
`conda install -n <env_name> requirements.txt`

**Comment by orduek** [comment]
Yes. Indeed this is the advantage of using a requierements.txt file. I think we can maybe make it a bit shorter, as many things are dependencies installed by the computer.

**Comment by orduek** [comment]
could you please remove the current req.txt file and generate a new one (maybe with fewer packages)? Currently, it generates a lot of vulnerabilities.
Thanks!

**Comment by cherruc** [comment]
Hi Or, I generated this from the one-time environment setup we did with:

```
module load miniconda
conda create -n py37_dev python=3.7 numpy scipy pandas matplotlib ipython jupyter
source activate env_name
conda install -c brainiak -c defaults -c conda-forge brainiak
conda install -c conda-forge nipype 
```

Which packages would not be necessary? Thanks!

**Comment by orduek** [comment]
have you tried using pip3 freeze instead of conda?
Take a look here
https://stackoverflow.com/questions/31684375/automatically-create-requirements-txt
